windows.js

```diff
@@ -4,11 +4,29 @@ import execall from 'execall';
 
 const TEN_MEBIBYTE = 1024 * 1024 * 10;
 
-export default async function killTabs() {
-	const command = 'wmic process where Caption=\'chrome.exe\' get CommandLine,ProcessId /format:list';
-	const response = await promisify(childProcess.exec)(command, {maxBuffer: TEN_MEBIBYTE});
+export default async function killTabs(options) {
+	const browsers = [];
 
-	if (response.stderr) {
+	if (options.chromium) {
+		browsers.push('Caption=\'chromium.exe\'');
+	}
+
+	if (options.chrome) {
+		browsers.push('Caption=\'chrome.exe\'');
+	}
+
+	if (options.brave) {
+		browsers.push('Caption=\'brave.exe\'');
+	}
+
+	if (options.edge) {
+		browsers.push('Caption=\'msedge.exe\'');
+	}
+
+	const args = ['process', 'where', browsers.join(' or '), 'get', 'CommandLine,ProcessId', '/format:list'];
```

Comment:
You overwrote https://github.com/sindresorhus/kill-tabs/pull/25/commits/8849b97bf03847dfdf924f0111c5089e0cc5d4e5

Reply:
oops I'm pushing a new commit to resolve it.